ci-operator/config/openshift/cluster-api-provider-ovirt/openshift-cluster-api-provider-ovirt-release-4.10.yaml

```diff
@@ -50,13 +50,13 @@ tests:
     cluster_profile: ovirt
     workflow: openshift-e2e-ovirt-minimal
 - as: e2e-ovirt-cleanup
-  interval: 4h
+  cron: 48 22 */7 * *
   steps:
     cluster_profile: ovirt
     test:
     - ref: openshift-e2e-ovirt-cleanup-test
 - as: e2e-ovirt-prfinder
-  interval: 5m
```

Comment:
as well

Reply:
Yeah - I've seen this one in the analytics data. They insist it is OK and it certainly doesn't stand up clusters. Let's preserve it.

Reply:
done, I kept also 4h interval in the job above as I feel is somehow connected. We can correct that in the 2nd run if needed.